index.html

```diff
@@ -0,0 +1,187 @@
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+         </div>
+         <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container">
+        <div> <img src="images/Portfolio_3.png" class="img_2"> </div>
+        <div class="section_1">
+         <h2> Tonic </h2>
+         <ul class="list-ul">
+           <li class="list_1"> CANOPY </li>
+           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+           <li class="list_2"><img src="images/Counter_2.png">2015</li>
+               </ul>
+         <p>A daily selection of privately
+            personalized reads; no accounts 
+            or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+         </div>
+         <button class="btn_2">See Project</button>
+     </div>
+
+     <div class="container">
+        <div> <img src="images/Portfolio_4.png" class="img_2"> </div>
+        <div class="section_1">
+         <h2 class="head_2"> Multi-Post <br> Stories </h2>
+         <ul class="list-ul">
+           <li class="list_1"> CANOPY </li>
+           <li class="list_2"><img src="images/Counter_1.png">Back End Dev</li>
+           <li class="list_2"><img src="images/Counter_2.png">2015</li>
+               </ul>
+         <p>A daily selection of privately
+            personalized reads; no accounts 
+            or sign-ups required.</p>   
+         <button class="btn_3"> html</button>
+         <button class="btn_4">css</button>
+         <button class="btn_5">javascript</button>
+         </div>
+        <button class="btn_2">See Project</button>
+      </div>
+
+  </div>
+</section>
+
+     <div class="container_2">
+        <section class = "section_2">
+          <h2> About <br> Myself</h2>
+          <p> I’m a software developer! I can help you build a product,
+            feature or website Look through some of my
+            work and experience! If you like what you see and
+             have a project you need coded, don’t hestiate to contact me.
+          </p>
+          <a href="#" class="connect_1"> LET'S CONNECT</a>
+          <ul class="social_2">
+            <li><a href="https://github.com/david30-maker"><img src="images/github1.png" alt="github logo"></a></li>
+            <li><a href="https://www.linkedin.com/in/david-igbo-464196184"><img src="images/Linkedin_1.png" alt="in logo"></a></li>
+            <li><a href="https://venture.angellist.com/v/begin"><img src="images/angellist.png" alt="angel logo"></a></li>
+            <li><a href="https://twitter.com/@okpahdavid"><img src="images/twitter_1.png" alt="Twiiter logo"></a></li>
+            <li><a href="#"><img src="images/medium.png" alt="m logo"></a></li>
+          </ul>
+        </section>
+    
+        <button class="btn"> Get my resume </button>
+    
+        <div class="div-7">
```

Comment:
- [ ] As suggested by the previous, The container for your language, framework and skills should also be a `ul` `li`.

- [ ] I kindly suggest that you also use a `ul` and create and `li` tag container instead of a div here

## EXAMPLE
```
   <ul>
        <li class="languague">
              <ul>
                  <li><a href="#">JavaScript</a></li>
                  <li><a href="#">HTML</a>
                  <li><a href="#">CSS</a>
             </ul>
         </li>
          <li class="framework">
              <ul>
                  <li><a href="#">Bootstrap</a></li>
                  <li><a href="#">Ruby on Rails</a>
                  <li><a href="#">RSPEC</a>
             </ul>
         </li>
            <li class="skills">
              <ul>
                  <li><a href="#">Code Kit</a></li>
                  <li><a href="#">GitHub</a>
                  <li><a href="#">Codepen</a>
             </ul>
         </li>                                     
</ul>
```

Please note that the above is just an example and may not apply directly to your code.